addons/main/config.cpp

```diff
@@ -541,26 +541,28 @@ class CfgPatches {
             "a3_weapons_f_vests",
             "a3data",
             "map_vr",
-            "extended_eventhandlers", "CBA_UI", "CBA_XEH", "CBA_XEH_A3"
+            "extended_eventhandlers", "CBA_UI", "CBA_XEH", "CBA_XEH_A3",
+            "ace_common"
         };
-        author[] = {"ACE Team"};
-        authorUrl = "";
-        versionDesc = "A.C.E.";
-        versionAct = QUOTE(['MAIN',_this] execVM '\z\ace\addons\main\about.sqf';);
+        author[] = {ELSTRING(common,ACETeam)};

```

Comment:
Pretty sure you'll need `ECSTRING` (difference from L is that it adds a `$` in front which is needed in configs).